android/src/main/java/com/google/samples/apps/ibeaconapp/LifecycleHandler.java

```diff
@@ -0,0 +1,86 @@
+package com.google.samples.apps.ibeaconapp;
+
+import android.app.Activity;
+import android.app.Application;
+import android.os.Bundle;
+import com.google.samples.apps.ibeaconapp.lightbluebean.IBeaconManager;
+
+public class LifecycleHandler implements Application.ActivityLifecycleCallbacks {
+    private int resumed;
```

Comment:
Really do not understand what is going on in this class.
Is it supposed to work in multithreads?  It won't work correct.
What if I push "Start", then "Pause", then "Resume" (looks like normal flow for this guy) - you won't start scanning process? ( resumed > paused = false)